scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -925,9 +934,12 @@
         (log/debug "Preparing to pull offer (offer-matcher)")
         (let [offer (async/<!! matcher-chan)
               fid (:fid offer)
+              offer-ids (:ids offer)
               driver (:driver offer)]
           (log/debug "(offer-matcher) Offer taken " offer)
-          (handle-resource-offer! conn driver fid pending-jobs-atom offer))
+          (if-let [invalid-offer-id (some #(cache/lookup @rescinded-offer-id-cache %) offer-ids)]
+            (log/info "Received rescinded offer" invalid-offer-id "in offer matcher")
```

Comment:
I'm not sure =)

On Fri, Mar 11, 2016 at 3:45 PM Li Jin <notifications@github.com> wrote:

> In scheduler/src/cook/mesos/scheduler.clj
> <https://github.com/twosigma/Cook/pull/102#discussion_r55886073>:
>
> >                driver (:driver offer)]
> >            (log/debug "(offer-matcher) Offer taken " offer)
> > -          (handle-resource-offer! conn driver fid pending-jobs-atom offer))
> > +          (if-let [invalid-offer-id (some #(cache/lookup @rescinded-offer-id-cache %) offer-ids)]
> > +            (log/info "Received rescinded offer" invalid-offer-id "in offer matcher")
>
> Isn't this the same as the rescinding rate?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twosigma/Cook/pull/102/files#r55886073>.
>